javascriptv3/example_code/lambda/scenarios/basic/README.md

```diff
@@ -0,0 +1,34 @@
+# Scenario - AWS Lambda for AWS SDK for JavaScript (v3)
+
+This is an interactive command prompt that showcases how the JavaScript SDK can be
+used for interacting with Lambda.
```

Comment:
This interactive command prompt shows you how to use the AWS SDK for JavaScript
with AWS Lambda.